javascript/javascript2/week1/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <meta charset="utf-8">
+  <title></title>
+</head>
+
+<body>
+  <script src="shortword-daletters.js"></script>
```

Comment:
Better a bit longer to make it more readable:
`short-words-danish-letters`

Reply:
Got it!